webmcp-analyzer/webmcp_analyzer/analyzer.py

```
@@ -0,0 +1,993 @@
+            ])
+
+        for sitemap_url in sitemap_urls:
+            try:
+                resp = self._session.get(sitemap_url, timeout=self.timeout)
+                if resp.status_code != 200:
+                    continue
+            except requests.RequestException:
+                continue
+
+            try:
+                root = ET.fromstring(resp.text)
```

Ruff (0.15.1)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)